--- a/src/it/violation-rule-threshold-check/src/main/java/com/test/Example.java	(revision c683034bc0fb2bd654d7d2b76e5763d2b91fdf24)
+++ b/src/it/violation-rule-threshold-check/src/main/java/com/test/Example.java	(date 1774712720505)
@@ -3,10 +3,10 @@
 public class Example {

 	public void sayHello(boolean a) {
-		if (a) {
-			System.out.println("if");
-		} else {
-			System.out.println("else");
+		if (a) {
+			System.out.println("if");
+		} else {
+			System.out.println("else");
 		}
 		System.out.println("return");
 	}
